lib/dirList.js

```diff
@@ -12,11 +12,11 @@ const dirList = {
    * @param {string} dotfiles
    * note: can't use glob because don't get error on non existing dir
    */
-  list: async function (dir, options, dotfiles) {
+  list: async function (dir, options, dotFiles) {
```

Comment:
My only comment would be to point out that "dotfile" is the correct nomenclature https://en.wikipedia.org/wiki/Hidden_file_and_hidden_directory#Unix_and_Unix-like_environments

Reply:
Fixed